pkg/volume/csi/csi_plugin.go

```diff
@@ -77,6 +77,12 @@ type driverMode string
 
 const persistentDriverMode driverMode = "persistent"
 const ephemeralDriverMode driverMode = "ephemeral"
+const combinedDriverMode driverMode = "persistent+ephemeral"
```

Comment:
I have missed most recent discussions, is this a new mode being introduced where driver can operate in ether mode ?

Reply:
Yes, exactly. KEP update is here: https://github.com/kubernetes/enhancements/pull/1129/files

Hostpath already implements that and supports both modes now: https://github.com/kubernetes-csi/csi-driver-host-path/pull/67

Reply:
These consts should be defined in the api. If the API is not there yet, then we should remember to remove these once we add it